pkg/deploy/tls.go

```diff
@@ -569,6 +569,27 @@ func SyncAdditionalCACertsConfigMapToCluster(deployContext *DeployContext) (bool
 		revisions += cm.ObjectMeta.Name + labelEqualSign + cm.ObjectMeta.ResourceVersion
 	}
 
+	isSelfSignedUsed, err := IsSelfSignedCertificateUsed(deployContext)
+	if err != nil {
+		return false, err
+	}
+	if isSelfSignedUsed {
+		selfSignedSecretName := types.NamespacedName{
+			Namespace: deployContext.CheCluster.Namespace,
+			Name:      CheTLSSelfSignedCertificateSecretName,
+		}
+		cheTLSSelfSignedCertificateSecret := &corev1.Secret{}
+		err = deployContext.ClusterAPI.Client.Get(context.TODO(), selfSignedSecretName, cheTLSSelfSignedCertificateSecret)
```

Comment:
Question to deploy team: can I assume that secret exists if isSelfSignedUsed is true?